draft-ietf-wish-whip.md

```diff
@@ -206,6 +208,8 @@ a=ice-pwd:0b66f472495ef0ccac7bda653ab6be49ea13114472a5d10a
 ~~~~~
 {: title="ICE restart request"}
 
+Given that in order to send new ICE candidetes to the WHIP resource, the WHIP client needs to know the entity-tag associated to the ICE session, it MUST buffer any gathered candidates before the HTTP response to the initial PUT request or the PATCH request with the new entity-tag value is received. Once the entity-tag value is know the WHIP client SHOULD send a single aggregated HTTP PATCH request with all the buffered ICE candidates.
```

Comment:
Id change the

> with all the buffered ICE candidates

with

> with all the ICE candidates it has buffered so far

as otherwise implementors may think you're asking them to wait for ALL candidates (thus making trickling useless).